hyperspy/tests/misc/test_compilers.c

```diff
@@ -0,0 +1,4 @@
+int main(void)
```

Comment:
yes it shoul be put to manifest, I noticed that, so what other place would be apropriate?